cfme/common/physical_server_views.py

```diff
@@ -158,10 +158,24 @@ class PhysicalServerManagePoliciesView(BaseLoggedInPage):
     save = Button("Save")
     reset = Button("Reset")
     cancel = Button("Cancel")
+    breadcrumb = BreadCrumb(locator='.//ol[@class="breadcrumb"]')
 
     @property
     def is_displayed(self):
-        return False
+        title = "'Physical Server' Policy Assignment"
+        return self.breadcrumb.active_location == title
+
+
+class PhysicalServerEditTagsView(BaseLoggedInPage):
+    """PhysicalServer's Manage Policies view."""
```

Comment:
**QUESTION:** Why does the class name and the docstring differ so much?   Shouldn't they have similar text?

Reply:
Yes, they should. I fixed this